cardano-cli/test/Test/OptParse.hs

```diff
@@ -1,3 +1,4 @@
+{-# LANGUAGE FlexibleContexts #-}
```

Comment:
What change made this needed?

Reply:
I've removed the pragma changes to see what happens in CI.

Reply:
Looks to be working.